ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/AttestationValidator.java

```diff
@@ -132,7 +125,9 @@ SafeFuture<InternalValidationResultWithState> singleOrAggregateAttestationChecks
             InternalValidationResultWithState.reject(
                 "Rejecting attestation because committee bits count is not 1"));
       }
+    }
 
+    if (attestation.isSingleAttestation() || attestation.requiresCommitteeBits()) {
```

Comment:
I would move this check to fail a little earlier in this method prob before we do all the checks that start on line 105